Core/Steps/ContinueReleaseStep.cs

```diff
@@ -56,8 +56,8 @@ public void Execute (string? ancestor, bool noPush)
     var currentBranchName = _gitClient.GetCurrentBranchName();
     if (string.IsNullOrEmpty(currentBranchName))
     {
-      const string message = "Could not continue the release because there was no current branch found.";
-      throw new InvalidOperationException(message);
+      const string message = "Could not identify the currently checked-out branch in the repository's working directory.";
+      throw new UserInteractionException(message);
```

Comment:
Based on the following assumption: there is no way during normal operations that a branch cannot be found. Alternative: this will happen whenever we are not in a git directory. In that case, the UIEx remains and the rest of that comment can be disregarded.

```suggestion
      throw new InvalidOperationException(message);
```
Please check all the 'GetCurrentBranchName' failed exceptions and switch to InvalidOpEx.